esrally/chart_generator.py

```diff
@@ -1320,6 +1488,14 @@ def __init__(self, track, cfg=None, challenge=None, car=None, node_count=None, c
     def name(self):
         return self.configuration.get("name")
 
+    @property
+    def flavor(self):
+        return self.configuration.get("flavor")
+
+    @property
+    def lic(self):
```

Comment:
What is `lic`? License? Is there a reason for the shortcut?

Reply:
Yes, it's license, but `license` is a built-in in Python: https://docs.python.org/3/library/builtins.html

Reply:
🤦‍♂️ - fair enough but does a property name really clash with a builtin? I mean, it's always used in the context of its enclosing object.

Reply:
It produces noise on the IDE at well and that was an additional motivation behind it. For variables at least it seems to be a [strong antipattern](https://stackoverflow.com/questions/16523789/naming-conflict-with-built-in-function) to shadow a built-in and @property in Python makes it act as an instance variable so didn't want to do this.

How about `eslicense`? Naming's not my strongest suit and yet it's hitting me all the time :)

Reply:
Maybe `es_license` then?